programs/vote/src/vote_processor.rs

```diff
@@ -771,7 +780,10 @@ mod tests {
             .convert_to_current();
         assert_eq!(
             vote_state.votes,
-            vec![Lockout::new(*vote.slots.last().unwrap())]
+            vec![LandedVote {
```

Comment:
we have this pattern repeated in several places. consider `impl From<Lockout> for LandedVote { ... }` so we can just `LandedVote::from(lockout)`

Reply:
> > Yes this can be done. Although it's not really possible to introduce new types behind a feature gate. At best, the usage of the types can be behind a feature.
> 
> the types themselves don't need to be gated, just their usage. best to keep them [minimally visible](https://doc.rust-lang.org/reference/visibility-and-privacy.html) to the rest of the code until the feature is fully introduced
> 
> > What is it I need to do to support this? Who do I coordinate with?
> 
> nothing more than choose a generic name for the vote state feature gate string. "vote state 1.15.x" or similar

I was asking about who I would coordinate to "may as well make room for epoch-locked commission changes while we're changing the vote state format".  I don't know if this means adding in more unused fields ready for use in a future change; or just saving extra space; in either case I would need more details.

Reply:
i was suggesting that you don't need to worry about it.  we can add the other stuff behind your gate once it lands

Reply:
sorry, unresolving.  missed that the comment i replied to was completely unrelated to the code comment